### PR TITLE
bwiernik new variables

### LINKED ISSUE #157: Proposal: Add "actor" name variable
Add actor/performer/cast member ("actor") name variable for multimedia formats such as broadcast and motion_picture. 

For the broadcast and motion_picture mediums, the performer is not also the author. For video (YouTube, etc) and song, the performer is also the author.

Example motion picture (CSL type motion_picture) MLA citation (using director and actor):
The Usual Suspects. Directed by Bryan Singer. Performances by Kevin Spacey, Gabriel Byrne, Chazz Palminteri, Stephen Baldwin, and Benicio Del Toro. Polygram, 1995. 

(http://columbiacollege-ca.libguides.com/mla/videos)

Example video (CSL type motion_picture) MLA citation (using author):
Sethi, Ramit. "How to Write a Winning Resume, With Ramit Sethi." YouTube, 23 June 2015, www.youtube.com/watch?v=_0fjkKCsM1w. Accessed 28 June 2016.

(http://columbiacollege-ca.libguides.com/mla/videos)

Adding an "author" name variable would simplify adding support for "Performance(s) by" prefix.

### PR DESCRIPTION
**Pull request by bdarcus**
Add name, date, string variables to schemas

Add available-date date variable.

Add chair, compiler, contributor, executive-producer, organizer,
producer, performer name variables.

Add archive_collection, division, part-title string variables.

Finally, pretty-prints csl-data.json using prettier.

Closes #157, #179, #163, #114

Addresses citation-style-language/zotero-bits#43
Addresses citation-style-language/zotero-bits#27
Addresses citation-style-language/zotero-bits#60
Addresses citation-style-language/zotero-bits#44
Addresses citation-style-language/zotero-bits#32
Addresses citation-style-language/csl-evolution#22

### CODE REVIEW

File: schemas/input/csl-data.json [hunk 1]
```diff
@@ -130,12 +154,30 @@
...
+                "items": {
+                    "$ref": "#/definitions/name-variable"
+                }
+            },
+            "performer": {
+                "type": "array",
+                "items": {
+                    "$ref": "#/definitions/name-variable"
+                }
+            },
```
Comment (rmzelle) [inline schemas/input/csl-data.json:426]: (minor nitpick, but "performer" and "producer" are not listed alphabetically)

File: schemas/input/csl-data.json [hunk 2]
```diff
@@ -289,18 +358,39 @@
             "original-title": {
                 "type": "string"
             },
-            "page": {
+            "part-title": {
```
Comment (rmzelle) [inline schemas/input/csl-data.json]: Should come after "page*" and "part" (alphabetical sorting)

File: schemas/styles/csl-variables.rnc [hunk 3]
```diff
@@ -22,15 +23,22 @@ div {
...
     | "director"
     | "editor"
     | "editorial-director"
+    | "executive-producer"
     | "illustrator"
     | "interviewer"
+    | "organizer"
     | "original-author"
+    | "producer"
+    | "performer"
```
Comment (rmzelle) [inline schemas/styles/csl-variables.rnc:41]: Idem.

File: schemas/input/csl-data.json (hunk 1 above)
Comment (bdarcus) [reply schemas/input/csl-data.json:426]: I'll fix these while I'm at it.

**Comment by bdarcus** [comment]
@bwiernik - can you please take a look at the first post, which I've revised with a proposed squash merge commit message. Basically, that would combine all of this into one commit.

I started to rebase, but realized it's probably not necessary.

**Comment by bwiernik** [comment]
Looks good.

**Pull request merged**